src/kvstore/DiskManager.cpp

```diff
@@ -102,6 +102,23 @@ StatusOr<PartDiskMap> DiskManager::partDist(GraphSpaceID spaceId) {
   return spaceIt->second;
 }
 
+void DiskManager::getDiskPaths(DiskPaths& diskPaths) {
+  std::lock_guard<std::mutex> lg(lock_);
```

Comment:
You could delete the `partDist` method if it is not used. BTW, add lock in every method in `DiskManager` except `refresh`.

Reply:
partDist is a old function , and I found it not used in anywhere, and delete it.